common/lib/src/main/java/org/apache/syncope/common/lib/to/AuditTO.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.syncope.common.lib.to;
+
+import org.apache.syncope.common.lib.BaseBean;
+
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.XmlType;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+@XmlRootElement(name = "audit")
+@XmlType
+public class AuditTO extends BaseBean implements EntityTO {
```

Comment:
This should be renamed after `AuditEntryTO`.
Also, I don't see reasons for implementing `EntityTO`, which should be reserved to transfer objects bound to JPA entities.

Reply:
This seems required by the logic API. All logic implementations require the type `<T extends EntityTO>`. Should we remove the concept of AuditLogic or is there an alternative to the internal API?

Reply:
mmm you're right, actually; let me resolve this item...